src/components/Team/index.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { FunctionComponent } from 'react'
+import { Col, Row } from 'reactstrap'
+import cn from 'classnames'
+
+import * as R from 'ramda'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faGithub, faLinkedinIn } from '@fortawesome/free-brands-svg-icons'
+import { IconProp } from '@fortawesome/fontawesome-svg-core'
+
+export enum Platforms {
+  github = 'github',
+  linkedIn = 'linkedIn',
+}
+export type Platform = Platforms
```

Comment:
Whats this for?

Reply:
Just a type alias for the enum object. Can be removed.